### DISCUSSION TITLE
Can a popup have a position set automatically?

### DISCUSSION THREAD

**Discussion by ppazos** [Leaflet/Leaflet#9315] [Q&A]
### Checklist

- [X] I've searched through the [plugins](https://leafletjs.com/plugins.html) to make sure this feature isn't already available, or think it shouldn't require a plugin.
- [X] I've searched through the [current issues](https://github.com/Leaflet/Leaflet/issues) to make sure this feature hasn't been requested already.
- [X] I agree to follow the [Code of Conduct](https://github.com/Leaflet/Leaflet/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.

### Motivation

All popups appear on top of markers and move the map. If the marker is at the top of the map, I would prefer the popup to appear under the marked and avoid the map to move.

### Suggested solution

Same as motivation.

**Comment by IvanSanchez** [comment]
Manually instantiate the popup, put it in a specific map pane, and disable the map handlers. The Leaflet documentation and examples should contain enough information for you to learn how to do all that.

**Reply by ppazos** [reply]
Hola @IvanSanchez is there an example on how to do that? I'm pretty new to this library. Thanks.

**Reply by IvanSanchez** [reply]
Read the documentation. Maybe you just need to disable auto-pan.
